src/components/Preview.css

```diff
@@ -63,6 +63,49 @@
   background: #484848;
 }
 
+.Frame-delete {
+  text-align: center;
+  margin-top: 3vw;
+}
+
+.Frame-redo {
+  text-align: center;
+  margin-top: 3vw;
+}
+
+.Frame-delete-redo-button {
+  width: 210px;
+  height: 35px;
+  margin-top: 10px;
+  font-size: 1em;
+  background: transparent;
+  border-radius: 5px;
+  color: #fff;
+  cursor: pointer;
+  outline: none !important;
+  animation: fade-in 0.5s;
+}
```

Comment:
buttons could use spacing in between, perhaps line 79 could go from :
`margin-top: 10px;` to `margin: 10px 10px 0;`